### DISCUSSION TITLE
PETG

### DISCUSSION THREAD

**Discussion by dhu77** [phnxdrgn536/FLSun-v400#7] [Ideas]
Hello

do you have a petg profile?

i test your pla profiles and is great !

Thanks

David

**Comment by phnxdrgn536** [comment]
i do not have any PETG profiles at this time. If you would like to forward me your settings you run at with PETG i can work on getting some set up.

**Comment by dhu77** [comment]
Ok, here is a PETG profile[](url)
[DHU_PETG_NORMAL 0 2.zip](https://github.com/phnxdrgn536/FLSun-v400/files/10356094/DHU_PETG_NORMAL.0.2.zip)

**Reply by phnxdrgn536** [reply]
what size nozzles do you run with this?

**Reply by phnxdrgn536** [reply]
added 0.4mm profiles for PETG

**Reply by dhu77** [reply]
0.4

**Reply by phnxdrgn536** [reply]
[flsun_v400_0.4_PETG.zip](https://github.com/phnxdrgn536/FLSun-v400/files/10356261/flsun_v400_0.4_PETG.zip)
try these

**Comment by dhu77** [comment]
Hello Joseph

I see the line width is 0.4 for the fast and ultra fast profiles, but the print is longer than the normal profile. By setting the width to 0.44 the print is much faster. This observation is valid for PLA and PETG profiles. what do you think ?

David
